lib/csv/table.rb

```diff
@@ -23,8 +23,19 @@ class Table
     # * length()
     # * size()
     #
-    def initialize(array_of_rows)
-      @table = array_of_rows
+    def initialize(array_of_rows, headers=nil)
```

Comment:
Since is a public class, can you add a documentation about the new parameter?

Reply:
I've added the documentation about headers argument.